### ISSUE TITLE
Atlas Docker Build compilation error

### ISSUE THREAD

**Issue by akhilreddyjirra** [odpi/egeria#1230]
when I ran mvn clean install -Ddocker -Ddocker.repo=XXX  for docker atlas. Giving me the below error.

```
[INFO] [INFO] Apache Atlas Notification 2.0.0-SNAPSHOT ........... SUCCESS [ 31.282 s]
[INFO] [INFO] Open Metadata support for Atlas 2.0.0-SNAPSHOT ..... FAILURE [02:48 min]
[INFO] [INFO] atlas-client-v2 2.0.0-SNAPSHOT ..................... SKIPPED
[INFO] [INFO] Apache Atlas UI 2.0.0-SNAPSHOT ..................... SKIPPED
[INFO] [INFO] Apache Atlas Web Application 2.0.0-SNAPSHOT ........ SKIPPED
[INFO] [INFO] Apache Atlas Documentation 2.0.0-SNAPSHOT .......... SKIPPED
[INFO] [INFO] Apache Atlas FileSystem Model 2.0.0-SNAPSHOT ....... SKIPPED
[INFO] [INFO] Apache Atlas Plugin Classloader 2.0.0-SNAPSHOT ..... SKIPPED
[INFO] [INFO] Apache Atlas Hive Bridge Shim 2.0.0-SNAPSHOT ....... SKIPPED
[INFO] [INFO] Apache Atlas Hive Bridge 2.0.0-SNAPSHOT ............ SKIPPED
[INFO] [INFO] Apache Atlas Falcon Bridge Shim 2.0.0-SNAPSHOT ..... SKIPPED
[INFO] [INFO] Apache Atlas Falcon Bridge 2.0.0-SNAPSHOT .......... SKIPPED
[INFO] [INFO] Apache Atlas Sqoop Bridge Shim 2.0.0-SNAPSHOT ...... SKIPPED
[INFO] [INFO] Apache Atlas Sqoop Bridge 2.0.0-SNAPSHOT ........... SKIPPED
[INFO] [INFO] Apache Atlas Storm Bridge Shim 2.0.0-SNAPSHOT ...... SKIPPED
[INFO] [INFO] Apache Atlas Storm Bridge 2.0.0-SNAPSHOT ........... SKIPPED
[INFO] [INFO] Apache Atlas Hbase Bridge Shim 2.0.0-SNAPSHOT ...... SKIPPED
[INFO] [INFO] Apache Atlas Hbase Bridge 2.0.0-SNAPSHOT ........... SKIPPED
[INFO] [INFO] Apache Atlas Kafka Bridge 2.0.0-SNAPSHOT ........... SKIPPED
[INFO] [INFO] Apache Atlas Distribution 2.0.0-SNAPSHOT ........... SKIPPED
[INFO] [INFO] ------------------------------------------------------------------------
[INFO] [INFO] BUILD FAILURE
[INFO] [INFO] ------------------------------------------------------------------------
[INFO] [INFO] Total time:  16:05 min
[INFO] [INFO] Finished at: 2019-07-02T04:20:48Z
[INFO] [INFO] ------------------------------------------------------------------------
[INFO] [ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.7.0:compile (default-compile) on project open-metadata: Compilation failure: Compilation failure: 
[INFO] [ERROR] /root/atlas/open-metadata/src/main/java/org/apache/atlas/openmetadata/adapters/repositoryconnector/LocalAtlasOMRSMetadataCollection.java:[407,36] cannot find symbol
[INFO] [ERROR]   symbol:   method getActiveTypesByWildCardName(java.lang.String,java.lang.String)
[INFO] [ERROR]   location: variable repositoryHelper of type org.odpi.openmetadata.repositoryservices.connectors.stores.metadatacollectionstore.repositoryconnector.OMRSRepositoryHelper
[INFO] [ERROR] /root/atlas/open-metadata/src/main/java/org/apache/atlas/openmetadata/adapters/repositoryconnector/AtlasRelationshipMapper.java:[551,64] cannot find symbol
[INFO] [ERROR]   symbol:   variable DATA_PLATFORM
[INFO] [ERROR]   location: class org.odpi.openmetadata.repositoryservices.connectors.stores.metadatacollectionstore.properties.instances.InstanceProvenanceType
[INFO] [ERROR] /root/atlas/open-metadata/src/main/java/org/apache/atlas/openmetadata/adapters/repositoryconnector/AtlasRelationshipMapper.java:[554,64] cannot find symbol
[INFO] [ERROR]   symbol:   variable EXTERNAL_ENGINE
[INFO] [ERROR]   location: class org.odpi.openmetadata.repositoryservices.connectors.stores.metadatacollectionstore.properties.instances.InstanceProvenanceType
[INFO] [ERROR] /root/atlas/open-metadata/src/main/java/org/apache/atlas/openmetadata/adapters/repositoryconnector/AtlasRelationshipMapper.java:[557,64] cannot find symbol
[INFO] [ERROR]   symbol:   variable EXTERNAL_TOOL
[INFO] [ERROR]   location: class org.odpi.openmetadata.repositoryservices.connectors.stores.metadatacollectionstore.properties.instances.InstanceProvenanceType
[INFO] [ERROR] /root/atlas/open-metadata/src/main/java/org/apache/atlas/openmetadata/adapters/repositoryconnector/AtlasEntityMapper.java:[682,64] cannot find symbol
[INFO] [ERROR]   symbol:   variable DATA_PLATFORM
[INFO] [ERROR]   location: class org.odpi.openmetadata.repositoryservices.connectors.stores.metadatacollectionstore.properties.instances.InstanceProvenanceType
[INFO] [ERROR] /root/atlas/open-metadata/src/main/java/org/apache/atlas/openmetadata/adapters/repositoryconnector/AtlasEntityMapper.java:[685,64] cannot find symbol
[INFO] [ERROR]   symbol:   variable EXTERNAL_ENGINE
[INFO] [ERROR]   location: class org.odpi.openmetadata.repositoryservices.connectors.stores.metadatacollectionstore.properties.instances.InstanceProvenanceType
[INFO] [ERROR] /root/atlas/open-metadata/src/main/java/org/apache/atlas/openmetadata/adapters/repositoryconnector/AtlasEntityMapper.java:[688,64] cannot find symbol
[INFO] [ERROR]   symbol:   variable EXTERNAL_TOOL
[INFO] [ERROR]   location: class org.odpi.openmetadata.repositoryservices.connectors.stores.metadatacollectionstore.properties.instances.InstanceProvenanceType
[INFO] [ERROR] -> [Help 1]
[INFO] [ERROR] 
[INFO] [ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[INFO] [ERROR] Re-run Maven using the -X switch to enable full debug logging.
[INFO] [ERROR] 
[INFO] [ERROR] For more information about the errors and possible solutions, please read the following articles:
[INFO] [ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
[INFO] [ERROR] 
[INFO] [ERROR] After correcting the problems, you can resume the build with the command
[INFO] [ERROR]   mvn <goals> -rf :open-metadata
[ERROR] The command '/bin/sh -c mvn clean install -DskipTests -Pdist,embedded-hbase-solr -DskipEnunciate -f ./atlas/pom.xml' returned a non-zero code: 1
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  35:50 min
[INFO] Finished at: 2019-07-02T04:20:49Z
```

**Comment by planetf1** [comment]
will let @cmgrote handle this. He has a tougher challenge with combining egeria code and atlas code together. May be worth considering whether any combined build (docker or other) should be an additional required check --

**Comment by cmgrote** [comment]
From the build steps, this looks like this is the Atlas fork (eg. `..../docker/atlas` rather than `..../docker/apache-atlas`)?  This is a fork that we don't plan to maintain going forward -- I understood that it was still working, but now be broken (?)

**Comment by planetf1** [comment]
Oops - I misread the name. I'll check

**Comment by akhilreddyjirra** [comment]
> From the build steps, this looks like this is the Atlas fork (eg. `..../docker/atlas` rather than `..../docker/apache-atlas`)? This is a fork that we don't plan to maintain going forward -- I understood that it was still working, but now may be broken (?)

@cmgrote  i am assuming `../docker/atlas` is not required instead apache-atlas will consider.

**Comment by planetf1** [comment]
The failing constants are defined in org.odpi.openmetadata.accessservices.dataplatform.utils - refactoring likely means atlas isn't pulling in all the needed dependencies

However as @cmgrote mentioned this atlas fork will be removed just as soon as we get some replacement UIs in place so we won't put too much effort into fixing. 

I'll take a short look at the egeria dependencies used by the fork

**Comment by planetf1** [comment]
Actually the first failure relates to getActiveTypesByWildCardName. This was removed in e1d1eb94c3974d75fe312f58c1f4dc667c043634 which was committed on June 30

It would be possible to refactor the connector, but I think we're fighting a losing battle as we want to move the egeria code on

@cmgrote @mandy-chessell @grahamwallis I think it's time to retire the old atlas connector
 - remove from the source
 - remove from the build infrastructure requirements 

We could see if old atlas image  will still work with the help charts but given @cmgrote 's work to build the new connector is progressing, and a merge on our container stuff is blocked whilst we supported old atlas, I'm inclined to say let's take it and move on.

It will impact consumers wanting to view what is in Egeria, we can individually talk to them and suggest using postman in the interim, with initial versions of new UIs expected in weeks

Ok?

If so I'll do the removal and merge
